packages/react-components/react-button/src/stories/Button/ButtonIcon.stories.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import { bundleIcon, CalendarMonthFilled, CalendarMonthRegular } from '@fluentui/react-icons';
-import { Tooltip } from '@fluentui/react-tooltip';
-import { Button } from '@fluentui/react-button';
+import { Tooltip, Button } from '@fluentui/react-components';
```

Comment:
_nit:_ Order these alphabetically

```suggestion
import { Button, Tooltip } from '@fluentui/react-components';
```